Desktop/components/JASP/Widgets/Supporters.qml

```diff
@@ -0,0 +1,104 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import JASP.Widgets
+
+
+Item
+{
+	height:							supportColumn.height
+	property int fontPixelSize:		14
+
+	ColumnLayout
+	{
+		id:							supportColumn
+		spacing:					welcomeRoot.scaler * 20
+		anchors.left:				parent.left
+		anchors.right:				parent.right
+
+		Text
+		{
+			text:					qsTr("JASP is supported by the following institutions:").replace(/, /g, ",&nbsp;").replace(/\n\n/g, "<br><br>")
+			font.family:			jaspTheme.font
+			font.weight:			Font.Normal
+			font.pixelSize:         fontPixelSize * welcomeRoot.scaler
+			color:					jaspTheme.white
+			wrapMode:				TextEdit.Wrap
+			renderType:				Text.QtRendering
+			textFormat:				Text.StyledText
+			horizontalAlignment:	Text.AlignHCenter
+			Layout.fillWidth:		true
+		}
+
+		SwipeView
+		{
+			id:						swipeSupporters
+			orientation:			Qt.Vertical
+			Layout.fillWidth:		true
+			clip:					true
+			currentIndex:			Math.floor(Math.random() * count)
+			wheelEnabled:			true
+
+			Timer
+			{
+				interval:			5000
```

Comment:
I would do a bit quicker 4 or even 3 seconds.

Reply:
Well, it starts on a different one every time and I already brought it down from 8s

Reply:
yeah you also don't want seizure-inducing changes